ddl/ddl_worker.go

```diff
@@ -415,26 +415,35 @@ func (w *worker) deleteRange(ctx context.Context, job *model.Job) error {
 	return errors.Trace(err)
 }
 
-// finishDDLJob deletes the finished DDL job in the ddl queue and puts it to history queue.
-// If the DDL job need to handle in background, it will prepare a background job.
-func (w *worker) finishDDLJob(t *meta.Meta, job *model.Job) (err error) {
-	startTime := time.Now()
-	defer func() {
-		metrics.DDLWorkerHistogram.WithLabelValues(metrics.WorkerFinishDDLJob, job.Type.String(), metrics.RetLabel(err)).Observe(time.Since(startTime).Seconds())
-	}()
-
+func jobNeedGC(job *model.Job) bool {
```

Comment:
It may be migrated into ```parser/model```. it is easy for others to reuse.

Reply:
It should be moved when other packages need it.